python/paddle/fluid/dygraph/dygraph_to_static/ast_utils.py

```diff
@@ -27,16 +27,64 @@
 
 from paddle.fluid import unique_name
 
-TRUE_FUNC_PRFIX = 'true_fn'
-FALSE_FUNC_PRFIX = 'false_fn'
+TRUE_FUNC_PREFIX = 'true_fn'
+FALSE_FUNC_PREFIX = 'false_fn'
+
+
+class IsControlFlowIfVisitor(gast.NodeTransformer):
+    """
+    Judge whether the node.test from Dygraph code dependent on paddle Tensor.
+    If does, it should satisfy:
+        1. must involve at least one var whose type is Tensor.
```

Comment:
Tiny English issues:

If you write period after number as list item such as ("1." , "2."), you should capitalize the first word since it is new sentence. If you don't like to capitalize, you can change "1." -> "1, "

So here you may change to:
    1. Must involve ... (this one is more common)
Or
    1, must involve ...

Reply:
Great advice! Got it. Thanks very much.